spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java

```diff
@@ -91,6 +97,14 @@ public void setCloseTimeout(int closeTimeout) {
 	}
 
 	/**
+	 * Set the operation timeout in seconds. Defaults to 30 seconds.
+	 * @param closeTimeout the timeout.
```

Comment:
I haven't polled it locally yet, but I know that this one is going to be red for my in the IDEA 

The real method param is `operationTimeout`, not `closeTimeout `

😄